styles/diagram/_elementLayerOwl.scss

```diff
@@ -0,0 +1,35 @@
+.ontodia-overlayed-owl_element {
```

Comment:
You don't need to copy every css class, with scss you can easily use existing ones and override only properties you need.